pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -2364,7 +2365,12 @@ public ByteBufPair newMessageAndIntercept(long consumerId, long ledgerId, long e
                 ackSet);
         ByteBufPair res = Commands.serializeCommandMessageWithSize(command, metadataAndPayload);
         try {
-            getBrokerService().getInterceptor().onPulsarCommand(command, this);
+            val brokerInterceptor = getBrokerService().getInterceptor();
+            if (brokerInterceptor != null) {
+                brokerInterceptor.onPulsarCommand(command, this);
+            } else {
```

Comment:
I don't think we need this else statement

Reply:
Since it is a debug thing, I hope it's ok to leave it there -- just in the unlikely case anyone was depending on an exception being raised. PLMK if you feel strongly otherwise.